docs/manual/manual_velocity_control_mechanisms.rst

```diff
@@ -0,0 +1,419 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_velocity_control_mechanisms:
+
+****************************************************
+Velocity and Voltage Based Control Mechanisms
+****************************************************
+
+===================================
+About Velocity and Voltage Control
+===================================
+Velocity and voltage control provide methods of commanding your Vertiq module to a specified rotational velocity. All Vertiq modules, 
+regardless of firmware style (speed or servo), can be controlled via velocity and voltage commands, though the 
+underlying implementation of control varies depending on the controllers available on your firmware. Both our 
+velocity and position controllers support three methods of setting velocity targets: Control Velocity, Control Voltage, 
+and Control PWM. All modules also support control via standard :ref:`Analog Hobby Protocols <manual_hobby>`, with options to map inputs to Control Velocity, Control Voltage, and Control PWM. The inputs are mapped by either the 
+:ref:`ESC Propeller Input Parser <esc_propeller_input_parser>` or :ref:`Servo Input Parser <servo_input_parser>`. Vertiq's speed modules 
+larger than our 23XX family can also be driven with :ref:`DroneCAN <manual_dronecan>`. 
+
+=========================================
+Velocity and Voltage Control Mechanisms
+=========================================
+Control Velocity
++++++++++++++++++++++
+Velocity Control Through :ref:`Propeller Motor Controller <propeller_motor_control>` v. :ref:`Multi Turn Angle Controller <multi_turn_control_label>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+All *Control Velocity* commands use closed loop PID control in order to control either a target velocity or position. 
+The controllers available on each module are firmware dependent. In order to find what controllers are available with your 
+module and firmware, please see your module's page available on the left-hand side of this page.
+
+Our :ref:`Propeller Motor Controller <propeller_motor_control>`, available only while using speed firmware, uses a velocity PID controller in which the input target is 
+a velocity, and error is calculated on the difference between the module's actual velocity and the target. 
+This is the default controller used when the module receives commands through :ref:`Analog Hobby Protocols <manual_hobby>` or 
+:ref:`DroneCAN <manual_dronecan>`. An important note is that while Hobby and DroneCAN commands are passed into the Propeller Motor Controller, the 
+exact type of control depends on the configured *mode* parameter. Please refer to these linked pages in order to learn more about  
```

Comment:
Two picky things:

I might link to here when you discuss the mode stuff, this has more direct coverage on what the mode means: https://iqmotion.readthedocs.io/en/feature-add_velocity_control_documentation/manual/manual_throttle.html#mode-maximums-and-direction

Also, I might say that "whether velocity control is used depends..." since this section is all about velocity control and to make clear why this is a relevant thing to mention here